ovos_config/config.py

```
@@ -13,45 +13,30 @@
 from os.path import isfile
 from typing import Optional
 
-from ovos_config.models import LocalConf, MycroftDefaultConfig, \
-    OvosDistributionConfig, MycroftSystemConfig, MycroftUserConfig, \
-    RemoteConf
-from ovos_config.locations import OLD_USER_CONFIG, get_xdg_config_save_path, \
-    get_xdg_config_locations
-from ovos_utils.file_utils import FileWatcher
+from ovos_config.locations import get_xdg_config_locations, ASSISTANT_CONFIG, USER_CONFIG
+from ovos_config.models import LocalConf, DefaultConfig, DistributionConfig, SystemConfig, AssistantConfig, \
+    MycroftDefaultConfig, OvosDistributionConfig, MycroftSystemConfig, MycroftUserConfig, RemoteConf
```

Ruff (0.8.2)

`ovos_config.models.MycroftDefaultConfig` imported but unused

Remove unused import

(F401)